libs/common/src/middleware/set-cache-control.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response, NextFunction } from 'express';
+
+/**
+ * middleware to set Cache-Control to no-store, no-cache, must-revalidate and max-age=0
+ * stops cached version of a page being rendered when going back to a page (using the browser back button)
+ * ensures the get controller for the page is called
+ * @param _req
+ * @param res
+ * @param next
+ */
+export const setCacheControl = (_req: Request, res: Response, next: NextFunction) => {
```

Comment:
Can we change this back to the old name? This is less descriptive and doesn't tell the reader the behaviour that is being set